From 48e7a26d9220ef21d5f5f623108cd9c647521c4f Mon Sep 17 00:00:00 2001
From: Valentina Fernandez <valentina.fernandezalanis@microchip.com>
Date: Wed, 22 Jun 2022 17:16:30 +0100
Subject: [PATCH 8/8] icicle: enable overlay support in defconfig

Add CONFIG_OF_LIBFDT_OVERLAY to icicle kit defconfigs to apply dts
overlays from U-boot

Signed-off-by: Valentina Fernandez <valentina.fernandezalanis@microchip.com>
---
 configs/microchip_mpfs_icicle_amp_defconfig | 1 +
 configs/microchip_mpfs_icicle_defconfig     | 1 +
 2 files changed, 2 insertions(+)

diff --git a/configs/microchip_mpfs_icicle_amp_defconfig b/configs/microchip_mpfs_icicle_amp_defconfig
index 920b8bbc14..f8dc3fba45 100644
--- a/configs/microchip_mpfs_icicle_amp_defconfig
+++ b/configs/microchip_mpfs_icicle_amp_defconfig
@@ -19,3 +19,4 @@ CONFIG_CMD_MTDPARTS=y
 CONFIG_MTDPARTS_DEFAULT="mtdparts=spi-nand0:8m(payload),128k(env),48m(fitimage)"
 CONFIG_USE_BOOTCOMMAND=y
 CONFIG_BOOTCOMMAND="run bootcmd_qspi;run bootcmd_mmc;run bootcmd_dhcp"
+CONFIG_OF_LIBFDT_OVERLAY=y
diff --git a/configs/microchip_mpfs_icicle_defconfig b/configs/microchip_mpfs_icicle_defconfig
index ba20e6b184..da0e948794 100644
--- a/configs/microchip_mpfs_icicle_defconfig
+++ b/configs/microchip_mpfs_icicle_defconfig
@@ -19,3 +19,4 @@ CONFIG_CMD_MTDPARTS=y
 CONFIG_MTDPARTS_DEFAULT="mtdparts=spi-nand0:8m(payload),128k(env),48m(fitimage)"
 CONFIG_USE_BOOTCOMMAND=y
 CONFIG_BOOTCOMMAND="run bootcmd_qspi;run bootcmd_mmc;run bootcmd_dhcp"
+CONFIG_OF_LIBFDT_OVERLAY=y
-- 
2.25.1

